loama/src/components/header/HeaderBase.vue

```diff
@@ -1,47 +1,40 @@
 <template>
     <header>
         <div class="wordmark">
-            <Loama />
+            <Loama/>
```

Comment:
this your eslint setting? Seems weird, most lint settings I know prefer `<Comp />` over `<Comp/>`

Reply:
ESLint doesn't complain about it, but when reformatting it, it does automatically apply it, so maybe it's a rule that is not enabled in the ESLint?

Reply:
if it auto applies its not a rule that is not enabled its probably another rule for this, would not worry too much was just wondering :)